exercises/sum-of-multiples/instructions.md

```diff
@@ -0,0 +1,27 @@
+# Instructions
+
+Your task is to write the code that calculates the energy points that get awarded to players when they complete a level.
+
+The points awarded depend on two things:
+
+- the number of the level that the player completed
+- the base value of each magical item collected by the player during that level
+
+The energy points are awarded according to the following rules:
+
+- For each magical item, take the base value and find all the multiples of that value that are less than or equal to the level number.
+- Combine the sets of numbers.
+- Remove any duplicates.
+- Calculate the sum of all the numbers that are left.
+
+Let's look an example;
+
+**The player completed level 20 and found two magical items with base values of 3 and 5.**
+
+To calculate the energy points earned by the player, we need to find all the unique multiples of these base values that are less than or equal to level 20.
+
+- Multiples of 3 up to 20: `{3, 6, 9, 12, 15, 18, 20}`
```

Comment:
```suggestion
- Multiples of 3 up to 20: `{3, 6, 9, 12, 15, 18}`
```

since 20 is not a multiple of 3, I think it is necessary to remove 20 from this list. The only other reason I could think of to include 20 is that the upper limit should always be included, but nothing seems to indicate that.

Reply:
You're absolutely correct.  This was totally just a messed up edit on my part. Thank you for catching it!